jquery-btvn.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function () {
+  let arrStudent = [
+    { name: "loc", point: 7 },
+    { name: "tien", point: 8 },
+    { name: "lin", point: 9 },
+    { name: "an", point: 8 },
+    { name: "can", point: 10 },
+    { name: "cuong", point: 6 },
+    { name: "cung", point: 5 }
+  ];
+
+  let tbody = $("tbody");
+  let btn = $("#btnshow");
+  let lengthArr = arrStudent.length;
+  let lengthCurrent = 2;
+
+  function showStudentInfo(){
+    let result = "";    
+    let i = 0;
+    for (i; i < lengthArr; i++) {
```

Comment:
```suggestion
    for (let i = 0; i < lengthArr; i++) {
```
biểu thức đầu nó chỉ chạy vào 1 lần, nên để thế này cũng đc e ạ